packages/jest/test/react-enzyme.test.js

```diff
@@ -3,14 +3,33 @@ import 'test-utils/enzyme-env'
 
 import jestInCase from 'jest-in-case'
 import * as enzyme from 'enzyme'
-import { css, jsx, ThemeProvider } from '@emotion/react'
+import {
+  __unsafe_useEmotionCache,
+  css,
+  jsx,
+  ThemeProvider,
+  EmotionCache
+} from '@emotion/react'
 import styled from '@emotion/styled'
 import React from 'react'
 import toJson from 'enzyme-to-json'
 
 import { matchers } from '@emotion/jest'
 import * as serializer from '@emotion/jest/enzyme-serializer'
 
+afterEach(() => {
+  let cache
+  function GetCache() {
+    cache = __unsafe_useEmotionCache()
+    return null
+  }
+  enzyme.shallow(<GetCache />)
+
+  cache.registered = {}
+  cache.inserted = {}
+  cache.sheet.flush()
+})
```

Comment:
this is a quick hack, I don't quite care about this - we'll be removing Enzyme's support (and thus this file) soon